README.md

```diff
@@ -16,8 +16,12 @@ Although the `flakehub-push` Action requires little configuration, you may benef
 
 ## Integration
 
-This action sets the `flakeref` output to the exact name and version that was published.
-The flake reference can be used in subsequent steps or workflows as part of a deployment process.
+This action sets outputs for integrating into CD pipelines:
+
+- `flake_name` Name of the flake. Example: `DeterminateSystems/flakehub-push`
+- `flake_version` Name of the flake. Example: `0.1.99+rev-2075013a3f3544d45a96f4b35df4ed03cd53779c`
```

Comment:
"Name of the flake" is repeated from above here

Reply:
Done!